hack/helm_vars/wire-server/values.yaml

```diff
@@ -15,10 +15,12 @@ tags:
   sftd: false
 
 cassandra-migrations:
+  imagePullPolicy: Always
```

Comment:
Why are these pullPolicy=always suddenly necessary if they were not before? This looks like an unrelated drive-by change.

Reply:
Yes, it is a drive by change. Earlier, this was not set because it couldn't be set, but for integration tests we should set imagePullPolicy to `Always` for all images that we create during integration test run.